python/dgl/nn/pytorch/conv/gatv2conv.py

```diff
@@ -0,0 +1,291 @@
+"""Torch modules for graph attention networks v2 (GATv2)."""
+# pylint: disable= no-member, arguments-differ, invalid-name
+import torch as th
+from torch import nn
+
+from .... import function as fn
+from ...functional import edge_softmax
+from ....base import DGLError
+from ..utils import Identity
+from ....utils import expand_as_pair
+
+# pylint: enable=W0235
+class GATv2Conv(nn.Module):
+    r"""
+
+    Description
+    -----------
+    Apply GATv2 from
+    `How Attentive are Graph Attention Networks?<https://arxiv.org/pdf/2105.14491.pdf>`__
+    over an input signal.
+
+    .. math::
+        h_i^{(l+1)} = \sum_{j\in \mathcal{N}(i)} \alpha_{i,j} W^{(l)} h_j^{(l)}
+
+    where :math:`\alpha_{ij}` is the attention score bewteen node :math:`i` and
+    node :math:`j`:
+
+    .. math::
+        \alpha_{ij}^{l} &= \mathrm{softmax_i} (e_{ij}^{l})
+
+        e_{ij}^{l} &= \vec{a}^T\mathrm{LeakyReLU}\left(W[h_{i} \| h_{j}]\right)
+
+    Parameters
+    ----------
+    in_feats : int, or pair of ints
+        Input feature size; i.e, the number of dimensions of :math:`h_i^{(l)}`.
+    out_feats : int
+        Output feature size; i.e, the number of dimensions of :math:`h_i^{(l+1)}`.
+    num_heads : int
+        Number of heads in Multi-Head Attention.
+    feat_drop : float, optional
+        Dropout rate on feature. Defaults: ``0``.
+    attn_drop : float, optional
+        Dropout rate on attention weight. Defaults: ``0``.
+    negative_slope : float, optional
+        LeakyReLU angle of negative slope. Defaults: ``0.2``.
+    residual : bool, optional
+        If True, use residual connection. Defaults: ``False``.
+    activation : callable activation function/layer or None, optional.
+        If not None, applies an activation function to the updated node features.
+        Default: ``None``.
+    allow_zero_in_degree : bool, optional
+        If there are 0-in-degree nodes in the graph, output for those nodes will be invalid
+        since no message will be passed to those nodes. This is harmful for some applications
+        causing silent performance regression. This module will raise a DGLError if it detects
+        0-in-degree nodes in input graph. By setting ``True``, it will suppress the check
+        and let the users handle it by themselves. Defaults: ``False``.
+    bias (bool, optional): If set to :obj:`False`, the layer will not learn
```

Comment:
"bias (bool, optional)" -> "bias: bool, optional"; Move "If ..." to the next line. This matters for building the documentation page.